static_report/package.json

```diff
@@ -81,7 +81,7 @@
     "bundle:case:comparison-asymmetric": "dotenv -e .env.comparison-asymmetric craco build",
     "bundle:e2e:norm": "run-p -l bundle:norm:*",
     "bundle:e2e:case": "run-p -l bundle:case:*",
-    "setup:e2e": "run-s embed:html:e2e extract:metadata lint:fix bundle:*",
+    "setup:e2e": "run-s embed:html:e2e extract:metadata bundle:*",
```

Comment:
The `html:e2e` this command does not exist, I think this command can be removed?

Reply:
Yes, please do. :) 
Forgot cleanup awhile back

Reply:
done